doc/source/manual/xtrabackup_sst.rst

```diff
@@ -59,27 +59,28 @@ For more information, see `socat(1)
 <http://www.dest-unreach.org/socat/doc/socat.html>`_.
 
 .. note:: Using ``transferfmt=nc`` does not support
-   any of the SSL-based encryption modes
-   (values ``2``, ``3``, and ``4`` for the :option:`encrypt` option).
-   Only ``encrypt=1`` is supported.
+   the SSL-based encryption mode 
+   (value ``4`` for the :option:`encrypt` option).
```

Comment:
The option names should be changed:
tca -> ssl-ca
tcert -> ssl-cert
tkey -> ssl-key